sklearn/model_selection/_search.py

```diff
@@ -587,6 +587,13 @@ def fit(self, X, y=None, groups=None, **fit_params):
                               'the "fit" method.', RuntimeWarning)
             else:
                 fit_params = self.fit_params
+        if self.return_train_score == "warn":
+            warnings.warn("Computing training scores is likely to affect "
```

Comment:
I don't think "is likely to" is honest. Usually, we'd hope/expect fitting to take much more time than scoring. But this is not always the case. "may affect" would be truer.